app/src/main/java/agency/five/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,65 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+
+
+data class ActorCardViewState(
+    val imageUrl: String?,
+    val name: String,
+    val character: String
+)
+@Composable
+fun ActorCard( actorCardViewState: ActorCardViewState,modifier: Modifier = Modifier)
+    {
+        Card(
+            modifier = modifier.padding(10.dp).width(130.dp).height(250.dp),
+            shape = RoundedCornerShape(20.dp),
+            elevation = 100.dp
+            ) {
+                Column {
+                        AsyncImage(
+                            model = actorCardViewState.imageUrl,
+                            contentDescription = actorCardViewState.name,
+                            modifier = modifier.height(170.dp),
+                            contentScale = ContentScale.Crop
+                        )
+                        Text(
+                            modifier = modifier.padding(top=8.dp).padding(horizontal = 15.dp).width(100.dp),
```

Comment:
keep in mind that text should have only horizontal padding. width here is redundant and text should go as much as it has space in the container - that means that you have to put fillMaxWidth to it so it keeps the width of the container - horizontal padding